packages/api/src/schema/api-schema.yaml

```diff
@@ -273,6 +273,44 @@ components:
         sessionId:
           type: string
           description: Session ID of the stream to clip
+    target:
```

Comment:
Is this different from the existing `multistream-target` that is referenced inside here? If so, in what way? Can you give this a clear name explaining why it exists and we're not just using the other definition? 

e.g. `multistream-target-input` would be already much clearer if it is a `multistream-target` to be received on request payload.